js/src/components/campaign-conversion-notice/conversion-statuses.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import { getNewPath } from '@woocommerce/navigation';
+import { Link } from '@woocommerce/components';
+import { LOCAL_STORAGE_KEYS } from '.~/constants';
+
+const BEFORE_CONVERSION = {
+	title: __(
+		'Your Google Listings & Ads campaigns will soon be automatically upgraded',
+		'google-listings-and-ads'
+	),
+	content: __(
+		'From July through September, Google will be upgrading your existing campaigns from Smart Shopping to Performance Max, giving you the same benefits, plus expanded reach. There will be no impact to your spend or campaign settings due to this upgrade.',
+		'google-listings-and-ads'
+	),
+	externalLink: {
+		link: '#',
+		content: __(
+			'Learn more about this upgrade',
+			'google-listings-and-ads'
+		),
+	},
+	localStorageKey: LOCAL_STORAGE_KEYS.IS_BEFORE_MIGRATION_NOTICE_DISMISSED,
+};
+
+const AFTER_CONVERSION = {
+	title: __(
+		'Your Google Listings & Ads campaigns have been automatically upgraded',
+		'google-listings-and-ads'
+	),
+	content: createInterpolateElement(
+		__(
+			'Google has auto-upgraded your existing campaigns from Smart Shopping to Performance Max, giving you the same benefits plus extended reach across the Google network. No changes were made to your campaign settings and metrics from previous campaigns will continue to be available in <link>Reports</link> for historical purposes.',
+			'google-listings-and-ads'
+		),
+		{
+			link: (
+				<Link
+					className="gla-campaign-conversion-status-notice-reports__link"
+					href={ getNewPath( null, '/google/reports' ) }
```

Comment:
Thanks, is the design team responsible to decide which links have an event recorded?

Reply:
> is the design team responsible to decide which links have an event recorded?

That was what I thought at the beginning since in my previous job the PM decides what events should be recorded. But from [WooCommerce's Usage Tracking](https://woocommerce.com/usage-tracking/) there is a Google Listings & Ads section saying what events we track:

> User behavior events during initial set up and general use of the extension such as when buttons are clicked and tabs or pages are viewed.

So I think developers can decide which events should be recorded as long as it's unique and we have written it down in the event JSDoc so the event tracking doc will be up to date. However it's just my understanding.

For this case, I think the link `Reports` in the migration notice may not be that crucial so I guess it's fine whether we record it or not.

Reply:
Can we please add events for both the links please? (both "Reports" and "Learn more about this upgrade") - I believe we have a [trackable link component](https://github.com/woocommerce/google-listings-and-ads/tree/develop/js/src/components) that might be useful for this.

Reply:
Thanks @jconroy's input! So I think going forward every link on the GLA and other extensions should be recorded as tracking events. 👌

Reply:
> every link on the GLA and other extensions should be recorded as tracking events.

I wouldn't go that far as blanket  rule of thumb  - In this case I think we might find it useful to see if merchants are looking for more info about what PMax is etc. and similarly I think it might be useful to see if merchans are wanting to check if their reports are still working